### PR TITLE
Create newsletter-april-2018.rst

### PR DESCRIPTION
**Pull request by kelly-o**

### CODE REVIEW

File: docs/blog/newsletter-april-2018.rst [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+########################################
+Write the Docs Newsletter – April 2018
+########################################
+
+Happy Monday! 
+Kelly O. here, with a ton of exciting community stuff to share with you this month! 
+There's a lot to cover, so I'll cut right to the chase!
+
+First, we're coming up on the Portland conference here in just four weeks (!?), and we're starting to get pretty stoked. 
+Tickets are now fully sold out, but if you're itching for a conference fix, now's a great time to look ahead to Prague in September... especially if you'd like to speak!
```
Comment (clairealyssa) [inline docs/blog/newsletter-april-2018.rst]: Tickets are fully sold out, but if you're itching for a conference fix, now's a great time to look ahead to Prague in September... especially if you'd like to speak! 

-- The two uses of "now" got a little confusing.

File: docs/blog/newsletter-april-2018.rst [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+----------------------------------------------------
+
+A question came up this month about the best approach for hosting the different parts – docs, examples, reference apps, and more – of a public SDK. 
+The heart of the original poster's question had to do with evolving their documentation approach as the product matured. 
+Developers began to contribute samples and reference apps, and it started to become apparent that the developer resources would need their own home and processes, separate from the user and admin content.
+
+The community came up with a handful of points that a good solution would need to address:
+
+* A plan for how developers will be able to update and maintain their samples from release to release.
+* Options for auto-generating reference documentation for the APIs, if possible
```
Comment (clairealyssa) [inline docs/blog/newsletter-april-2018.rst]: add a period at the end of this bullet

File: docs/blog/newsletter-april-2018.rst [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+*Got a finicky formatting or mechanics question? The friendly folks in* `#general <https://writethedocs.slack.com/messages/general>`_ *can help you answer it.*
+
+------------------------------------------
+New tool to try out: automated screenshots
+------------------------------------------
+
+Although we don't often feature conversations about specific tools, there was one that popped up this month that we thought was worth a quick mention. 
+On what feels like a weekly basis, someone brings up the difficulty of keeping screenshots up to date. 
+So when someone pointed to `CronShot <https://www.npmjs.com/package/cronshot>`_, a Node module for programmatically scheduling and saving screenshots of web pages, we thought we'd share it around. 
+In addition, if you're so inclined (and comfortable with JavaScript), you can add the `Nick.js <https://www.npmjs.com/package/nickjs>`_ scraping library (oh my!) in order to automate browser navigation altogether.
```
Comment (clairealyssa) [inline docs/blog/newsletter-april-2018.rst]: If a short explanation is even possible, it might be nice to add one for what scraping is in this context.

**Review by clairealyssa — COMMENTED** [review]
I have minimal feedback, and so I say both "congratulations" and "I'm sorry". ;)

File: docs/blog/newsletter-april-2018.rst (hunk 3 above)
Comment (kelly-o) [reply docs/blog/newsletter-april-2018.rst]: a good call – I went the other direction, though, and eliminated the ref to scraping altogether

**Pull request merged**